src/docfx/config/EnvironmentVariable.cs

```diff
@@ -21,6 +21,11 @@ public static class EnvironmentVariable
 
         public static string? SessionId => GetValue("DOCFX_SESSION_ID");
 
+        public static readonly DocsEnvironment DocsEnvironment =
```

Comment:
same here, try to limit `DOCS_` within `OpsXyz`

Reply:
These can all be kept in `OpsAbc`, just one issue, how can `RegressionTest` use the `PubDev graph client`

Reply:
regression test docfx config would override ops accessor